libpod/networking_linux.go

```diff
@@ -238,7 +252,15 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
 					continue
 				}
 				if status.Exited() {
-					return errors.New("slirp4netns failed")
+					// Seek at the beginning of the file and read all its content
+					if _, err := logFile.Seek(0, 0); err != nil {
+						logrus.Errorf("could not seek log file: %q", err)
+					}
+					logContent, err := ioutil.ReadAll(logFile)
+					if err != nil {
+						return errors.New("slirp4netns failed")
```

Comment:
errors.Wrapf(err, "slirp4netns failed")

Reply:
Or at least slirp4netns failed, unable to read output of slirp4netns command.

Reply:
fixed